zer0-os/zOS
# fix(check-icons-on-last-message): handle admin message and received message scenarios

### What does this do?
- fixes issue where check icons are not rendering for last message if last message is an admin message or a received message.

### Why are we making this change?
- to ensure check icons are rendered on the last message sent by the current user.

### How do I test this?
- run tests as usual.
- run ui > send a message > ensure check icon appears on the last message you sent.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
  
  
  
<img width="912" alt="Screenshot 2024-06-06 at 17 26 12" src="https://github.com/zer0-os/zOS/assets/39112648/ca9f5231-ed70-40e4-a5f2-33e30724f5b1">
